wiki/History_of_osu!/April_Fools/en.md

```diff
@@ -0,0 +1,324 @@
+---
+tags:
+  - april fools
+  - april 1st
+  - 1 april
+  - april 1
+  - april fools day
+  - joke
+  - history
+  - legacy
+---
+
+# List of osu! April Fools' day jokes
+
+*For the complete history of osu!, see: [History of osu!](/wiki/History_of_osu!)*
+
+Every year, the osu! developers like to play pranks on the community on April Fools' day. This article contains a list of all the April Fools' day jokes that were played on the osu! community since 2009.
```

Comment:
```suggestion
Every year, the [osu!team](/wiki/People/The_Team) likes to play pranks on the community on April Fools' day. This article contains a list of all the April Fools' day jokes that were played on the osu! community since 2009.
```